src/allowed_args.cpp

```diff
@@ -721,6 +721,7 @@ static void addElectrumOptions(AllowedArgs &allowedArgs)
         .addArg("electrum.port", requiredStr, "Port electrum RPC listens on (default: mainnet 50001, testnet: 60001")
         .addArg("electrum.host", requiredStr, "Host electrum RPC listens on (default: 127.0.0.1)")
         .addArg("electrum.addr.limit", requiredStr, "Max txs to look up per address (default: 500)")
+        .addArg("electrum.shutdownonerror", optionalBool, "Shutdown Bitcoin Unlimited if the electrum server exits unexpectedly")
```

Comment:
To make forking and rebranding easier, we've removed all casual references to our name.  IDK if it matters much anymore, but we might as well keep doing it.  I think in this case its obvious what's being shut down so you could just say "Shutdown if electrum server exits unexpectedly".  Other times use use the executable name "bitcoind".